src/main/java/io/reactivex/observers/BaseTestConsumer.java

```diff
@@ -565,9 +565,14 @@ public final U assertValuesOnly(T... values) {
     }
 
     /**
-     * Assert that the TestObserver/TestSubscriber received only the specified values in any order.
-     * <p>This helps asserting when the order of the values is not guaranteed, i.e., when merging
+     * Assert that the TestObserver/TestSubscriber received only items that are in the specified
+     * collection as well irrespective of the order they were received.
+     * <p>
+     * This helps asserting when the order of the values is not guaranteed, i.e., when merging
      * asynchronous streams.
+     * <p>
+     * To ensure that only the expected items have been received, no more and no less, in any order,
+     * apply {@link #assertValueCount(int)} with {@code expected.size()}.
```

Comment:
I guess ideally method name would indicate that by using a word like `contains`, maybe add it to 3.x list of changes?

Reply:
`assertValuesIn(Collection<? extends T> expected)`?

Reply:
Posted #6153.